config/.env

```diff
@@ -0,0 +1,8 @@
+# Location of WordPress core develop directory
+WP_DEVELOP_DIR=/app/public/core-git/
```

Comment:
It doesn't seem that `WP_DEVELOP_DIR` is used? It seems like plugins in particular use it in their test bootstrapping, is that right? They'll suffer from `.env` not persisting in this case and will need to manually set the environment variable when running `phpunit`.

Reply:
Yeah, I only set it because there is no fixed standard for tests whether to use `WP_TESTS_DIR` or `WP_DEVELOP_DIR`.

I'd say since this is a (hopefully temporary) hack, we don't _really_ need to add the other variable to the `phpunit` command. However, I don't have a strong opinion about this, we might as well set both.